### DISCUSSION TITLE
Why don't the resource page work? How should I get the template?

### DISCUSSION THREAD

**Discussion by shiro6x** [elrumo/macOS_Big_Sur_icons_replacements#1496] [Help with the website]
<img width="1196" alt="Screenshot 2024-08-07 at 6 40 54 AM" src="https://github.com/user-attachments/assets/02bc6775-3d48-4559-a139-bbf6585716f5">
<img width="1470" alt="Screenshot 2024-08-07 at 6 41 30 AM" src="https://github.com/user-attachments/assets/0ace2f04-1257-4746-b6e2-dd84e568e67a">

**Comment by vikasbhatia** [comment]
I had downloaded it from the site before it was having issues; here you go:
[macOS App Icon Template.psd.zip](https://github.com/user-attachments/files/16551785/macOS.App.Icon.Template.psd.zip)

@spiny-lumpsucker @stoatsnhoney mentioning you guys too in case you were looking for this, since you commented in #1482.
